src/main/java/executor/task/FixedDuration.java

```diff
@@ -2,13 +2,15 @@
 
 public class FixedDuration extends Task {
 
+    String input;
```

Comment:
Please make the declaration explicit. If it's supposed to be private, do indicate.

Reply:
Also, is this property really necessary?